Content.Server/Chat/ChatSystem.cs

```diff
@@ -330,6 +340,24 @@ private void SendInVoiceRange(ChatChannel channel, string message, string messag
         _chatManager.ChatMessageToMany(channel, message, messageWrap, source, hideChat, sessions.Select(s => s.ConnectedClient).ToList());
     }
 
+    /// <summary>
+    ///     Input a string, outputs the correct tone that you can get with Loc.GetString
+    /// </summary>
+
+    private string GetToneBySuffix(string s)
+    {
+        var LOCtone = "chat-manager-entity-say-wrap-message";
+
+        foreach(var tone in _toneSuffixes) {
+            if (s.EndsWith(tone.Key)) {
+                LOCtone = tone.Value;
+                break;
+            }
+        }
+
+        return LOCtone;
+    }
```

Comment:
Summary isn't attached.

Also think it might be better as a regex?

Reply:
guh i had to learn regex 101 for this. probably worth it